docs/lang/articles/reference/global_settings.md

```diff
@@ -11,9 +11,15 @@ sidebar_position: 3
   `ti.init(device_memory_GB=0.5)`.
 - To specify which GPU to use for CUDA:
   `export CUDA_VISIBLE_DEVICES=[gpuid]`.
+- To specify which GPU to use for VULKAN:
+- `export TI_VISIBLE_DEVICE=[gpuid]`
 - To disable a backend (`CUDA`, `METAL`, `OPENGL`) on start up, e.g. CUDA:
   `export TI_ENABLE_CUDA=0`.
 
+:::note
+
+Under the case that you may want to use cuda as taichi kernel backend, and vulkan as GGUI backend. If you have multiple GPU devices, then you should take care of if `CUDA_VISIBLE_DEVICES` is matched with `TI_VISIBLE_DEVICE`. In general, `CUDA_VISIBLE_DEVICES` and `TI_VISIBLE_DEVICE` should indicate the same uuid of a GPU device. Use `nvidia-smi -L` to quiery the details of your GPU devices.
```

Comment:
Under the hood, GGUI uses Vulkan to perform rendering. Therefore, in the scenario where CUDA is used as the primary  backend, one should avoid setting different `uuid`s for `CUDA_VISIBLE_DEVICES` and `TI_VISIBLE_DEVICE`, so as to guarantee correct data transfer between CUDA and Vulkan devices. You may use `nvidia-smi -L` to query the details of your GPU devices